### PR TITLE
[github] Add CODEOWNERS for translations

### PR DESCRIPTION
**Pull request by rtsisyk**
Members of relevant translation teams will be notified and auto-assigned
to all PRs of their languages.

Signed-off-by: Roman Tsisyk <roman@tsisyk.com>

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+android/res/values-el/ @organicmaps/translations-el
+android/res/values-es/ @organicmaps/translations-es
+android/res/values-eu/ @organicmaps/translations-eu
+android/res/values-fa/ @organicmaps/translations-fa
+android/res/values-fi/ @organicmaps/translations-fi
+android/res/values-fr/ @organicmaps/translations-fr
+android/res/values-fr-rCA/ @organicmaps/translations-fr
+android/res/values-he/ @organicmaps/translations-he
+android/res/values-hu/ @organicmaps/translations-hu
+android/res/values-in/ @organicmaps/translations-id
```
Comment (vng) [inline .github/CODEOWNERS:16]: -in

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+android/res/values-it/ @organicmaps/translations-it
+android/res/values-ja/ @organicmaps/translations-ja
+android/res/values-ko/ @organicmaps/translations-ko
+android/res/values-nl/ @organicmaps/translations-nl
+android/res/values-nb/ @organicmaps/translations-no
```
Comment (vng) [inline .github/CODEOWNERS:21]: -nb

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+android/res/values-zh-rMO/ @organicmaps/translations-zh
+android/res/values-zh-rTW/ @organicmaps/translations-zh
+iphone/Maps/LocalizedStrings/ar.lproj/ @organicmaps/translations-ar
+iphone/Maps/LocalizedStrings/be.lproj/ @organicmaps/translations-be
+iphone/Maps/LocalizedStrings/bg.lproj/ @organicmaps/translations-bg
+iphone/Maps/LocalizedStrings/cs.lproj/ @organicmaps/translations-cs
+iphone/Maps/LocalizedStrings/da.lproj/ @organicmaps/translations-da
+iphone/Maps/LocalizedStrings/de.lproj/ @organicmaps/translations-de
+iphone/Maps/LocalizedStrings/el.lproj/ @organicmaps/translations-el
+iphone/Maps/LocalizedStrings/es-MX.lproj/ @organicmaps/translations-es
```
Comment (vng) [inline .github/CODEOWNERS:45]: add regular es

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+iphone/Maps/LocalizedStrings/fi.lproj/ @organicmaps/translations-fi
+iphone/Maps/LocalizedStrings/fr.lproj/ @organicmaps/translations-fr
+iphone/Maps/LocalizedStrings/he.lproj/ @organicmaps/translations-he
+iphone/Maps/LocalizedStrings/hu.lproj/ @organicmaps/translations-hu
+iphone/Maps/LocalizedStrings/id.lproj/ @organicmaps/translations-id
+iphone/Maps/LocalizedStrings/it.lproj/ @organicmaps/translations-it
+iphone/Maps/LocalizedStrings/ja.lproj/ @organicmaps/translations-ja
+iphone/Maps/LocalizedStrings/ko.lproj/ @organicmaps/translations-ko
+iphone/Maps/LocalizedStrings/nl.lproj/ @organicmaps/translations-nl
+iphone/Maps/LocalizedStrings/nb.lproj/ @organicmaps/translations-no
```
Comment (vng) [inline .github/CODEOWNERS:57]: nb

File: .github/CODEOWNERS [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+iphone/Maps/LocalizedStrings/pl.lproj/ @organicmaps/translations-pl
+iphone/Maps/LocalizedStrings/pt-BR.lproj/ @organicmaps/translations-pt
```
Comment (vng) [inline .github/CODEOWNERS:59]: add regular pt

**Review by vng — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (rtsisyk) [reply .github/CODEOWNERS:16]: Indonesian has `in` code on Android and `id` code on iOS:

- https://github.com/championswimmer/android-locales
- https://gist.github.com/jacobbubu/1836273

File: .github/CODEOWNERS (hunk 4 above)
Comment (rtsisyk) [reply .github/CODEOWNERS:57]: Updated.

File: .github/CODEOWNERS (hunk 3 above)
Comment (rtsisyk) [reply .github/CODEOWNERS:45]: Added

File: .github/CODEOWNERS (hunk 2 above)
Comment (rtsisyk) [reply .github/CODEOWNERS]: Updated

**Review by vng — APPROVED** [review]

**Pull request merged**